.gitignore

```diff
@@ -114,3 +114,6 @@ dist
 .vscode-test
 
 # End of https://www.toptal.com/developers/gitignore/api/node
+/.vs
+/api/csharp/.vs/pubsub-publish
```

Comment:
This is already covered by the `/api/csharp/.gitignore`

Reply:
My fault, that .gitignore shouldn't have been changed... will move it to /api/csharp/.gitignore